rpc/handlers.go

```diff
@@ -52,9 +193,138 @@ func (s *rpcServer) GetV1ConsensusState(ctx echo.Context) error {
 	})
 }
 
+// Queries
+
+func (s *rpcServer) PostV1QueryAccount(ctx echo.Context) error {
+	var body QueryAddressHeight
+	if err := ctx.Bind(&body); err != nil {
+		return ctx.String(http.StatusBadRequest, "bad request")
+	}
+
+	// Get latest stored block height
+	currentHeight := int64(s.GetBus().GetConsensusModule().CurrentHeight())
+	if currentHeight > 0 {
+		currentHeight -= 1
+	}
+	height := body.Height
+	if height == 0 {
+		height = currentHeight
+	}
+	readCtx, err := s.GetBus().GetPersistenceModule().NewReadContext(currentHeight)
+	if err != nil {
+		return ctx.String(http.StatusInternalServerError, err.Error())
+	}
+	accBz, err := hex.DecodeString(body.Address)
+	if err != nil {
+		return ctx.String(http.StatusBadRequest, err.Error())
+	}
+	amount, err := readCtx.GetAccountAmount(accBz, height)
+	if err != nil {
+		return ctx.String(http.StatusInternalServerError, err.Error())
+	}
+
+	return ctx.JSON(http.StatusOK, Account{
+		Address: body.Address,
+		Coins:   []Coin{{Amount: amount, Denom: "upokt"}},
+	})
+}
+
+func (s *rpcServer) PostV1QueryAccounts(ctx echo.Context) error {
+	var body QueryHeightPaginated
+	if err := ctx.Bind(&body); err != nil {
+		return ctx.String(http.StatusBadRequest, "bad request")
+	}
+
+	// Get latest stored block height
+	currentHeight := int64(s.GetBus().GetConsensusModule().CurrentHeight())
+	if currentHeight > 0 {
+		currentHeight -= 1
+	}
+	height := body.Height
+	if height == 0 {
+		height = currentHeight
+	}
+	readCtx, err := s.GetBus().GetPersistenceModule().NewReadContext(currentHeight)
+	if err != nil {
+		return ctx.String(http.StatusInternalServerError, err.Error())
+	}
+	allAccounts, err := readCtx.GetAllAccounts(height)
+	if err != nil {
+		return ctx.String(http.StatusInternalServerError, err.Error())
+	}
+
+	start, end, totalPages, err := getPageIndexes(len(allAccounts), int(body.Page), int(body.PerPage))
+	if err != nil && !errors.Is(err, errNoItems) {
+		return ctx.String(http.StatusBadRequest, err.Error())
+	}
+	if totalPages == 0 || errors.Is(err, errNoItems) {
+		return ctx.JSON(http.StatusOK, QueryAccountsResponse{})
+	}
+
+	accounts := make([]Account, 0)
+	for _, account := range allAccounts[start : end+1] {
+		accounts = append(accounts, Account{
+			Address: account.Address,
+			Coins:   []Coin{{Amount: account.Amount, Denom: "upokt"}},
+		})
+	}
+
+	return ctx.JSON(http.StatusOK, QueryAccountsResponse{
+		Result:     accounts,
+		Page:       body.Page,
+		TotalPages: int64(totalPages),
+	})
+}
+
+func (s *rpcServer) PostV1QueryAccounttxs(ctx echo.Context) error {
```

Comment:
```suggestion
func (s *rpcServer) PostV1QueryAccountTxs(ctx echo.Context) error {
```

Reply:
This is annoying as all the paths `/v1/query/accounttxs` are lowercase we cannot capitalise besides the first word without changing the path to have the capitalised letter. Not sure if this is alright but am going to put that in and we can change back if not desirable